test/functional/test_framework/test_node.py

```diff
@@ -226,17 +235,19 @@ def wait_for_rpc_connection(self):
                     'bitcoind exited with status {} during initialization'.format(self.process.returncode)))
             try:
                 rpc = get_rpc_proxy(
-                    rpc_url(self.datadir, self.index, self.chain, self.rpchost),
+                    self.init_rpc_url,
                     self.index,
                     timeout=self.rpc_timeout // 2,  # Shorter timeout to allow for one retry in case of ETIMEDOUT
                     coveragedir=self.coverage_dir,
                 )
-                rpc.getblockcount()
+                rpc.consensus_best_block_id()
                 # If the call to getblockcount() succeeds then the RPC connection is up
                 if self.version_is_at_least(190000):
```

Comment:
I think some of this code could be simplified if you removed these version checks. I don't think `190000` is a meaningful version number for `mintlayer-core` at this time so IMO it's just adding unnecessary noise. I would add it back when we have a version `190000` of the node and it needs to do this thing with mempool.

Reply:
So the amount of version checks is not all that massive after all. Removed.